config/initializers/okcomputer.rb

```diff
@@ -0,0 +1,37 @@
+require 'okcomputer'
+
+OkComputer.mount_at = 'status' # use /status or /status/all or /status/<name-of-check>
+OkComputer.check_in_parallel = true
+OkComputer::Registry.deregister 'database' # we don't have a database
+
+##
+# REQUIRED checks
+
+# Simple echo of the VERSION file
+class VersionCheck < OkComputer::AppVersionCheck
+  def version
+    File.read(Rails.root.join('VERSION')).chomp
+  rescue Errno::ENOENT
+    raise UnknownRevision
+  end
+end
+OkComputer::Registry.register 'version', VersionCheck.new
+
+##
+# EXTERNAL Services
+
+OkComputer::Registry.register 'external-solr', OkComputer::SolrCheck.new(Dor::SearchService.solr.uri.to_s)
+
+# Simple check to ping the Fedora server by asking it to describe the repository
+class FedoraCheck < OkComputer::Check
+  def check
+    conn = ActiveFedora::Base.connection_for_pid(0)
+    profile = conn.repository_profile # use vs. `profile` to force a GET call to the server for `/fedora/describe`
+    mark_message "Connected to #{profile['repositoryName']} #{profile['repositoryVersion']} via Rubydora #{Rubydora::VERSION}"
+  rescue => e
+    mark_failure
+    mark_message "Failure: #{e.class.name}: #{e.message}"
+  end
+end
```

Comment:
a bit of history: the Fedora check was a built-in `rubydora` check in `is_it_working-cbeer` developed circa 2012. The above `FedoraCheck` is a simplified version that forces a GET call to `/fedora/describe` on each check.

@eefahy confirmed that we are not doing app-level Fedora checks via Nagios right now due to lack of Nagios support for certificate-based authentication (she's looking into that further). So, as far as I know, the `rubydora` checks are the only monitoring of Fedora that we do. We'd like to move to a situation where Nagios is doing direct monitoring of Fedora.